### ISSUE TITLE
Project dead?

### ISSUE THREAD

**Issue by Unknow-max** [google/material-design-icons#1054]
As title, Is the project dead?

**Comment by CrlsMrls** [comment]
yes please, could maintainers clarify this? files are not being updated, the files served in https://fonts.googleapis.com/icon?family=Material+Icons **differ quite a lot** from the ones in this repository.

**Comment by Unknow-max** [comment]
The most beauty thing in that situation? That Google use this icons in all her projects, but no One care About the repo itself. Fantastic 😂😂😂👋👋👏

**Comment by rsheeter** [comment]
**Tl;dr** Project is alive, we will be bringing this repo up to date, I'm not currently sure of the timeline.

Context: Icons recently joined Google Fonts and we inherited this repository with it. We recently ran a survey (thank you to everyone who took time to fill it out) to help us decide if we should maintain it or just declare it deprecated. We concluded that we need to maintain the repo. We are now trying to figure out the best way to do that and how to prioritize that effort against various other projects.
